Automattic/wp-calypso
# Migrate extensions CSS styles to webpack

Some extensions' style were imported in the global `style.scss`, bloating the initial load for everyone, some extensions have a `css` field in their section definition and use the legacy CSS-loading code.

This PR migrates them all to use ESM style imports and the webpack pipeline.

**How to test:**
On your testing Jetpack/Atomic site, check the plugin UI for Woocommerce store, Zoninator (Zone Manager) and WP Super Cache. Verify that their UI is correctly styled.
